content/data/moderndatawarehouse/readme.md

```diff
@@ -1,35 +1,33 @@
-# Welcome to Microsoft Learn Live </br> Presented by FastTrack for Azure <br/> 
+# Welcome to FastTrack for Azure Live </br> Modern Data Warehouses - Designing, Building and Operating
+## (Or how I learned to stop worrying and just build a MDW)
+---
 ## We will start 1-2 minutes after the scheduled time to accommodate those still connecting
 
-<!--This call will not be recorded due to the wide audience and to encourage questions.-->
+This call will not be recorded due to the wide audience and to encourage questions.
 
 **Questions?** Feel free to type them in the chat window at any time. Note that questions you post will be public.
 
-<!--**Slideless** No PowerPoint, we promise! -->
+**Slideless** No PowerPoint, we promise!
 As we update this content ([aka.ms/ftalive-mdw](aka.ms/ftalive-mdw)) you will get the changes straight away.
 
-<!--**Feedback** Please give us your feedback on https://aka.ms/ftalive-feedback-->
-**Feedback** Please give us your feedback on https://aka.ms/ftalearnlive
+**Feedback** Please give us your feedback on https://aka.ms/ftalive-feedback
+<!--**Feedback** Please give us your feedback on https://aka.ms/ftalearnlive-->
 
-<!--**Prerequisites**
+**Prerequisites**
 You will need:
 * Microsoft Teams desktop client or Teams web client to view presentation.
 * Optional headphones to make it easier to listen to and interact with the presenters.
--->
-</br>
 
-# Modern Data Warehouses - Designing, Building and Operating
-## Or how I learned to stop worrying and just build a MDW
 ## Who we are
 **FastTrack for Azure (FTA) ([aka.ms/fta](aka.ms/fta))** <br/>
-Presenter: Ben Harding ([linkedin.com/in/ben-harding-fta](linkedin.com/in/ben-harding-fta)) <br/>
+<!--Presenter: Ben Harding ([linkedin.com/in/ben-harding-fta](linkedin.com/in/ben-harding-fta)) <br/>
 Presenter: Nitin Mehta ([linkedin.com/in/nitinmht](linkedin.com/in/nitinmht)) <br/>
 Moderator: Miho Yamamoto ([linkedin.com/in/mihoyamamoto](linkedin.com/in/mihoyamamoto))<br/>
 Moderator: Osamu Hirayama ([linkedin.com/in/osamuhir](linkedin.com/in/osamuhir))
```

Comment:
you need to add a break (<br/>) at the end of this line for formatting

Reply:
Fixed